input/script_utils_test.go

```diff
@@ -559,14 +559,14 @@ func TestHTLCSenderSpendValidation(t *testing.T) {
 // incoming HTLC.
 //
 // The following cases are exercised by this test:
-//  * receiver spends
-//     * HTLC redemption w/ invalid preimage size
-//     * HTLC redemption w/ invalid sequence
-//     * HTLC redemption w/ valid preimage size
-//  * sender spends
-//     * revoke w/ sig
-//     * refund w/ invalid lock time
-//     * refund w/ valid lock time
+//   - receiver spends
+//   - HTLC redemption w/ invalid preimage size
+//   - HTLC redemption w/ invalid sequence
+//   - HTLC redemption w/ valid preimage size
+//   - sender spends
```

Comment:
Need to fix the indentation of the second-level items in this file.

Reply:
cool fixed, well sort of. Seems like the new formatter really don't like nested list.

> Rewriting the text to avoid nested lists usually improves the documentation and is the best solution. Another potential workaround is to mix list markers, since bullet markers do not introduce list items in a numbered list, nor vice versa.